build.gradle

```diff
@@ -17,20 +17,28 @@ plugins {
     id 'com.gradle.build-scan' version '1.0'
 }
 
-apply plugin: 'java'
-group = 'org.mockito'
-description = 'Core API and implementation.'
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+description = 'Mockito mock objects library core API and implementation'
 
-apply plugin: 'maven-publish'
+apply from: 'gradle/root/version.gradle'
 
-apply from: 'gradle/version.gradle'
-apply from: "gradle/ide.gradle"
-apply from: 'gradle/coverage.gradle'
-apply from: 'gradle/osgi.gradle'
-apply from: 'gradle/gradle-fix.gradle'
-apply from: 'gradle/license.gradle'
+ext {
+    artifactName = 'mockito-core'
+    bintrayAutoPublish = true
+    mavenCentralSync = true
+}
+
+apply from: "gradle/java-library.gradle"
+archivesBaseName = 'mockito-core'
+apply from: "gradle/publishable-java-library.gradle"
+
+apply from: 'gradle/root/ide.gradle'
+apply from: 'gradle/root/coverage.gradle'
+apply from: 'gradle/root/gradle-fix.gradle'
+
+apply from: 'gradle/mockito-core/osgi.gradle'
+apply from: 'gradle/mockito-core/javadoc.gradle'
+apply from: 'gradle/mockito-core/license.gradle'
+apply from: 'gradle/release.gradle'
```

Comment:
So every `apply from:` after `archivesBaseName = 'mockito-core'` are applied for the `mockito-core` ?